paymentsheet/src/test/java/com/stripe/android/paymentsheet/viewmodels/PrimaryButtonUiStateMapperTest.kt

```diff
@@ -322,6 +324,37 @@ class PrimaryButtonUiStateMapperTest {
         }
     }
 
+    @Test
+    fun `Formats currency correctly based on per app localization`() = runTest {
```

Comment:
Can you add another test where the applications locales is null/empty list? Or add an assert where get locales is an empty list.

Reply:
Done 👍